rest_live/consumers.py

```diff
@@ -269,3 +269,34 @@ def model_saved(self, event):
             self.send_broadcast(
                 subscription.request_id, model_label, action, instance_data, renderer
             )
+
+    def model_deleted(self, event):
+        channel_name: str = event["channel_name"]
+        instance_pk: int = event["instance_pk"]
+        model_label: str = event["model"]
+
+        viewset_class = self.registry[model_label]
+
+        for subscription in self.subscriptions[channel_name]:
+            view = viewset_class.from_scope(
+                subscription.action,
+                self.scope,
+                subscription.view_kwargs,
+                subscription.query_params,
+            )
+            renderer = view.perform_content_negotiation(view.request)[0]
+            is_existing_instance = instance_pk in subscription.pks_in_queryset
+
+            if is_existing_instance:
+                instance_data = {
+                    view.lookup_field: instance_pk,
```

Comment:
I don't think it's a correct assumption that the `lookup_field` is always `pk`. For `penn-courses`, the `Section` APIView has the `lookup_field` be the section ID rather than the model's primary key, like `CIS-120-001` would be the lookup key.

I think the way to get around this would be to store the `lookup_field` inside `pks_in_queryset`, probably would have to turn it into a dictionary from a set in the progress of that.

Either way, would also probably be good to add a test case where `lookup_field` is a field other than the primary key. Documentation here: https://www.django-rest-framework.org/api-guide/generic-views/